fetch/cross-origin-resource-policy/resources/hello.py

```diff
@@ -0,0 +1,6 @@
+def main(request, response):
+    headers = [("Cross-Origin-Resource-Policy", request.GET['corp'])]
+    if 'origin' in request.headers:
+        headers.append(('Access-Control-Allow-Origin', request.headers['origin']))
```

Comment:
I assume you're going to use this bit in a future test that checks the `no-cors` restrictions?

Reply:
Yeah, this file is from #11171. I copied it here so the tests can run standalone and I don't have to include all prior commits making this harder to review.